pom.xml

```diff
@@ -16,14 +16,19 @@
     <modules>
         <module>ojp-adapter</module>
         <module>ojp-adapter-integration</module>
```

Comment:
(not sure that _integration_ is worldwide understood as integration test. Integration is also putting together several systems. eventually `ojp-adapter-int-test`. )

Reply:
(will be renamed in other PR)